src/util/tr.js

```diff
@@ -499,6 +496,29 @@ const _tr = (msg, {
       const body = "<div class='wzrkPPdscr' style='color:" + textColor + "'>" + descriptionText + '<div></td></tr></table></div>'
       html = css + title + body
     }
+    if (displayObj.preview && displayObj['custom-editor']) {
+      iframe.sandbox = 'allow-scripts allow-popups allow-popups-to-escape-sandbox'
+      /* eslint-disable no-cond-assign */
+      var properties = {}
+      var styleRegex = /<style[^>]*>([^<]*)<\/style>/g
+      var ctClassRegex = /\.CT_(?:Box|Banner)\s*{([^}]*)}/g // Regex to match either .CT_Box or .CT_Banner
+      var cssRegex = /([\w-]+)\s*:\s*([^;}\s]+)(?:;|$)/g // Regex to extract property-value pairs
+      var styleContent, cssBlock, cssMatch
+
+      // Extract style blocks
+      while (styleContent = styleRegex.exec(html)) {
+        // Extract the .CT_Box or .CT_Banner CSS block
+        while (cssBlock = ctClassRegex.exec(styleContent[1])) {
+          // Extract property-value pairs
+          while (cssMatch = cssRegex.exec(cssBlock[1])) {
```

Comment:
this does not look right to me, can you explain what are you doing here?

Reply:
- 'ctClassRegex' is used to find the CSS blocks for .CT_Box or .CT_Banner within the style content.
- 'cssRegex' extracts individual CSS properties from the matched CSS block (like height: 500px;).

The two nested while loops allow the script to:
Identify and iterate over each relevant CSS block (.CT_Box or .CT_Banner).
Extract and process each CSS property within those blocks, looking specifically for the height property.

Reply:
@PraveenCTzen This can be optimized, add terminating conditions as well.

Reply:
@KambleSonam Can you help clarify the type of terminating statements we should add here?

Reply:
I meant at what condition will the loop break?

Reply:
I have updated the condition of the while loop. Please review it.